docs/apis/core/dml/delegated-transactions.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Delegated database transactions
+tags:
+  - core_dml
+  - DML
+  - core
+  - API
+---
+
+## General principles
+
+1. No matter of the nested nature, they are really **"delegated transactions"** (so outer levels get the control over inner ones), so we are going to apply that naming to enforce the real concept
```

Comment:
I have no idea what this sentence means. I know this is a document migration, but we're trying to take the opportunity to update documentation at the same time to make it clearer and more meaningful.

Reply:
I've removed this line. I don't know what it means either.